packages/contracts-bedrock/test/setup/Setup.sol

```diff
@@ -175,9 +177,10 @@ contract Setup {
 
     /// @dev Sets up the L2 contracts. Depends on `L1()` being called first.
     function L2() public {
-        console.log("Setup: creating L2 genesis, with output mode %d", uint256(l2OutputMode));
+        console.log("Setup: creating L2 genesis with fork %d", uint256(l2Fork));
```

Comment:
Should this be `l2Fork.toString()`?

Reply:
yes, overlooked when switching to `toString()` 🙏🏻